### DISCUSSION TITLE
Add a switch to blog posts to enable "by-works" patronage from subscribers

### DISCUSSION THREAD

**Discussion by tshiunghan** [funmusicplace/mirlo#198] [Ideas]
Re: [How to compete with Patreon](https://siderea.dreamwidth.org/1824441.html)

Maybe not a priority, but it seems pretty straightforward to implement.

**Comment by simonv3** [comment]
Wouldn't this make more sense for it to be based on albums?

**Reply by tshiunghan** [reply]
Can an album be greater than or equal to one track?

**Reply by simonv3** [reply]
There's no reason an album couldn't be one track but that's interesting, what does the progressive release of an album look like

**Comment by simonv3** [comment]
It looks like the way to do this would be with [SetupIntents](https://stripe.com/docs/api/setup_intents)
